<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaMenuAgregarPerfil</class>
 <widget class="QMainWindow" name="ventanaMenuAgregarPerfil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1900</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1900</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Agregar un perfil</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelFondoMenuAgregarPerfil">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>1901</width>
      <height>850</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>menuAgregarPerfilFigma.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="labelConfiguracion">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>0</y>
      <width>531</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelAgregarPerfil">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>390</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNombrePerfil">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSalarioMinimo">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>420</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSalarioMaximo">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>460</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxAreaTrabajo">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Administracion / Oficina</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Almacen / Logistica / Transporte</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Atencion a clientes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Call center / Telemercado</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Compras / Comercio exterior</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Construccion / Obra</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Contabilidad / Finanzas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Direccion / Gerencia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Diseño / Artes graficas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Docencia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hosteleria / Turismo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Informatica / Telecomunicaciones</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ingenieria</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Investigacion y calidad</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Legal / Asesoria</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mantenimiento y reparacioens tecnicas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medicina / Salud</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mercadotecnia / Publicidad / Comunicacion</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Operarios / Produccion / Manufactura</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Recursos humanos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Servicios generales</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aseo y seguridad</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Otros</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEditPuestoTrabajo">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxIdioma">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>530</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ingles</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Frances</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Portugues</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Italiano</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aleman</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No aplica</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxNivelIdioma">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>570</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Basico</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Intermedio</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Avanzado</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No requiere un idioma adicional</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxEscolaridad">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Educacion básica o secundaria</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Educacion media superior o bachillerato</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Educacion superior o licenciatura</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxUbicacion">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>540</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Aguascalientes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Baja california</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Baja california sur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Campeche</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chiapas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chihuahua</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Coahuila</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Colima</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hidalgo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jalisco</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Durango</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Michoacan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Morelos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nayarit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nuevo leon</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Oaxaca</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Puebla</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Queretaro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quintana roo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>San luis potosi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sinaloa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sonora</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tabasco</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tamaulipas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tlaxcala</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Veracruz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yucatan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Zacatecas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ciudad de mexico</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Estado de mexico</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Guerrero</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Guanajuato</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelVolverInicio">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>431</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxAnosExperiencia">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>620</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Sin experiencia requerida</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 años de experiencia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 años de experiencia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9 años de experiencia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12 años de experiencia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15 años de experiencia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18 años de experiencia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21 años de experiencia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24 años de experiencia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>27 años de experiencia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30 años de experiencia</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelNombrePerfil">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Nombre del perfil:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPuestoDeseado">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Puesto deseado:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAreaFuncional">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Área funcional:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEscolaridad">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Escolaridad:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSueldoMensualMinimo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Sueldo mensual mínimo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSueldoMensualMaximo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>470</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Sueldo mensual máximo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSector">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>510</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Sector o giro:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSector">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>500</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelHerramientasDominadas">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>290</y>
      <width>191</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Herramientas dominadas:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHerramientas">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>290</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelHabilidadesTecnincas">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>330</y>
      <width>191</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Habilidades técnicas:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHabilidadesTecnicas">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>330</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelHabilidadesBlandas">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>370</y>
      <width>191</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Habilidades blandas:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHabilidadesBlandas">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>370</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelUbicacion">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>550</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Ubicación:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLogrosDestacados">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>410</y>
      <width>191</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Logros destacados:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLogros">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>410</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelCertificaciones">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>460</y>
      <width>191</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Certificaciones:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCertificaciones">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>450</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelDisponibilidad">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>500</y>
      <width>191</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Disponibilidad:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxDisponibilidad">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>490</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Disponibilidad inmediata</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sin disponibilidad</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelIdioma">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>540</y>
      <width>191</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Idioma:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNivelIdioma">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>580</y>
      <width>191</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Nivel de idioma:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAnosExperiencia">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>630</y>
      <width>191</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Años de experiencia</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCarrera">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>590</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Carrera afín:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCarrera">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>580</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelEdad">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>630</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Edad:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdad">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>620</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelInformacionExtra">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>680</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Información adicional:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditInformacionAdicional">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>660</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitulacion">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>680</y>
      <width>191</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(48, 55, 132);</string>
    </property>
    <property name="text">
     <string>Titulación requerida:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTitulacion">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>670</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Titulado</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Titulación no requerida</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
